tools/setupResourceScripts/setup_tdr_resources.py

```diff
@@ -274,6 +277,14 @@ def create_snapshots(clients, dataset_name, snapshots, profile_id):
     return snapshot_ids
 
 
+def create_snapshot_request(clients, snapshot_id):
+    snapshot_access_request = load_file("snapshot-access-request.json")
+    snapshot_access_request['sourceSnapshotId'] = snapshot_id
```

Comment:
Can you reformat this file with `black`? I would expect this to use `"`s and black will do that, along with other things.